src/stores/NotificationsController.ts

```diff
@@ -26,12 +27,12 @@ export const subscribeToNotifications = async (uuid: string = '') => {
           case NotificationsTypes.ACCOUNT_LOGOUT:
             await logout();
             if (notification.resource?.reason === 'sessions_limit') {
-              window.location.href = addQueryParams(window.location.href, { u: 'simultaneous-logins' });
+              window.location.href = addQueryParams(window.location.href, { u: 'login' });
```

Comment:
It looks like the simultaneous-logins message param got moved... was that intentional?

Reply:
No it was mistake 4b1070fd500fadb5b4f62390f3c44b332d18b7f8